_build_cfg.yml

```diff
@@ -34,7 +34,7 @@ Distros: openshift-enterprise,openshift-dedicated
 Topics:
   - Name: Overview
     File: index
-  - Name: OpenShift v2 vs. OpenShift v3
+  - Name: Comparing OpenShift Enterprise 2 and OpenShift Enterprise 3
```

Comment:
Sorry, I meant to mention yesterday also a suggestion to move this topic to the end of the Release Notes category vs the beginning. Or really, would this better belong in the Architecture category? Seems it would go well right after Architecture -> Overview, right before the Infrastructure Components subcategory. WDYT? Was there a request somewhere about it being specifically in the Rel Notes section?